packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -530,6 +539,16 @@ class CupertinoTextField extends StatefulWidget {
   /// and [CupertinoColors.activeOrange] in the dark theme.
   final Color cursorColor;
 
+  /// Controls how tall the selection highlight boxes are computed to be.
+  ///
+  /// See [ui.BoxHeightStyle] for details on available styles.
+  final ui.BoxHeightStyle selectionHeightStyle;
```

Comment:
(unrelated to this PR, so please ignore) I wish the enum doc had actual images showing off the difference.